src/main/resources/lib/guillotine/generic/generic-types.js

```diff
@@ -524,6 +524,9 @@ function createGenericTypes(context) {
             aggregationsAsJson: {
                 type: graphQlLib.Json,
             },
+            highlightAsJson: {
+                type: graphQlLib.Json,
```

Comment:
I guess thats the only way :)

I did the same

https://github.com/enonic/app-patchman/blob/952317ed2ce7c90d27c54545368e6b7c811d7951/src/main/resources/admin/tools/patchman/graphql/queries/addMultiRepoQuery.ts#L278

https://github.com/enonic/lib-explorer/blob/de5e6ca590213c71c20c3d03beedfb146a171425/src/main/resources/lib/explorer/interface/graphql/output/addOutputFieldsSearchResultHit.ts#L21

https://github.com/enonic/app-explorer/blob/3365ef9fed0c797c1e7c1c9579f1f222dd6cec96/src/main/resources/services/graphQL/document/addQueryDocuments.ts#L711

https://github.com/enonic/app-explorer/blob/52302cc45788110dbf6fb032231a94ed2fb4c8f6/src/main/resources/services/graphQL/synonym/addSynonymTypes.ts#L107